### PR TITLE
Link checker

### PR DESCRIPTION
**Pull request by chrismatheson**
sitedesk/issues#993

I've made a couple of changes to remove internal links that referenced `#` which was causing the linter problems and seemed to not affect the pages when removed. but it might be worth someone else having a whack at these on other browsers.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -42,13 +42,10 @@
             <li class='name'>
               <h1>
                 <a href='/'>
-                  <img src='{{site.baseurl}}/images/sitedesk/logo.png' />
+                  <img src='{{site.baseurl}}/images/sitedesk/logo.png' alt="logo" />
                 </a>
               </h1>
             </li>
-            <li class='toggle-topbar menu-icon'>
-              <a href='#'>Menu</a>
```
Comment (chrismatheson) [inline _layouts/default.html]: this wasn't visible so i assumed it wasn't required, however CSS and multi browser might be worth checking out

File: _includes/video.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <div class="videoButton">
-   <a href="#" data-reveal-id="videoModal" style="position:relative">
```
Comment (chrismatheson) [inline _includes/video.html]: removing hash href here didn't break playing of video

**Comment by chrismatheson** [comment]
@drmattsuu could you please check that the video on https://sitedesk.xyz/features-technical-benefits/ works on  IE11, Edge and Android 🙏 

I've tested on iOS and seems to be fine their

**Comment by drmattsuu** [comment]
@chrismatheson Can confirm - works on IE, edge and android (default and chrome browsers)

**Comment by chrismatheson** [comment]
https://sitedesk.slack.com/archives/general/p1462886282000037

**Pull request merged**